### ISSUE TITLE
House Consumption

### ISSUE THREAD

**Issue by Logzy72** [mletenay/home-assistant-goodwe-inverter#80]
Im wondering if this intergration reads the house consumption from the inverter if the inverter has a goodwe smart meter connected to it via RS485.
I have a MS series inverter and i dont see any fields for house consumption data.
This could be because the smart meter i have is the Goodwe Homekit which is basically the Goodwe smart meter with a wifi module attached to it.
 but i would have thought Id still see the sensor form consumption but the value would be 0.
When my solar was installed the installer did not connect the RS485 from the meter to the inverter.
If I was to connect the RS485 link I think the Goodwe Homekit would then send the consumption data to the inverter.

Could this intergration then read that consumption data from the inverter.

**Comment by Logzy72** [comment]
What do I need to do to see the xx* sensors?

**Comment by jebeke65** [comment]
The GW5000S-BP has it also in the plug-in but the value it produces is much too high, hence not correct. When I check the value during the night, I’m getting around 1500W which is certainly not what is consumed in the house

**Comment by midadami** [comment]
I have asked GoodWe and plan to keep asking until I get an answer as to why DT based inverters (DT, D-NS, XS, MS) don't offer the house consumption value through SolarGo app. It's probably because they made it that way and need firmware update or similar. My D-NS does have access to house consumption as excess electricity is activated and that can only happen if inverter has access to house consumption number on the moment.

**Comment by stevoh6** [comment]
> I've just release v0.9.3 where I re-enabled the xx sensors, so you should now see them out of the box.
> Most of them don't provide any values, but e.g. xx134, xx142, xx144 seem to report something, we just need to figure out what.

How and where can I see those values? I have shelly and GW DT inverter with smartmeter, so I can play with it, but I need some basic instructions. Im able to run Python or Nodejs scripts to read registers.

**Comment by teranex** [comment]
> > The basic formula is power produced by PV + power retrieved from battery (if any) - power exported to grid (measured by smart meter). All these input values are provided directly by the inverter.
> 
> So it does not include the power imported from the grid?
> 
> And the difference between House Consumption and Load is, that Load also contains the battery charge?

@bruxy70  Did you ever find an answer to what the difference between Load and House Consumption is? I'm also wondering about it, but can't really find an answer anywhere.

**Comment by bruxy70** [comment]
No, I did not. It is confusing. Based on all the explanations, I think it is supposed to be the same in my scenario (since I have no other load than the house consumption). But it is not.

**Comment by teranex** [comment]
> No, I did not. It is confusing. Based on all the explanations, I think it is supposed to be the same in my scenario (since I have no other load than the house consumption). But it is not.

Yeah I would also expect that in my case as I also have no other load. The only thing I could imagine would be that House Consumption would be what the house uses, and Load might be House Consumption + Power usage of the inverter itself. But I don't think that is the case since I noticed during the day the House Consumption is always higher than the Load (by 50 - 100W), but House Consumption is below Load during the night...

**Comment by AT622CZ** [comment]
Hi, we need update basic formula for House Consuption to  "power produced by PV + power retrieved from battery (if any) + **power retrieved from battery 2 (if any)** - power exported to grid. Will be possible?  (Batt2 power register: 35264)
